js/oceanex.js

```diff
@@ -0,0 +1,323 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError } = require ('./base/errors');
+
+module.exports = class oceanex extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'oceanex',
+            'name': 'oceanex',
+            'countries': [ 'SG' ],
+            'urls': {
+                'api': 'https://api.oceanex.pro/v1/',
+                'www': 'https://www.oceanex.pro.com',
+                'doc': 'https://api.oceanex.pro/doc/v1/',
+            },
+            'api': {
+                'public': {
+                    'get': [
+                        'markets',
+                        'tickers/{pair}/',
+                        'tickers_multi',
+                        'order_book',
+                        'order_book/multi',
+                        'fees/trading',
+                        'trades',
+                        'timestamp',
+                    ],
+                },
+                'private': {
+                    'get': [
+                        'key',
+                        'members/me',
+                        'orders',
+                        'orders_filter',
+                    ],
+                    'post': [
+                        'orders',
+                        'orders/multi',
+                        'order/delete',
+                        'order/delete/multi',
+                        'orders/clear',
+                    ],
+                },
+            },
+            'version': 'v1',
+            'rateLimit': 1000,
```

Comment:
Can you elaborate a bit on the actual rate limits? Because, in the exchange api docs we see varying limits, from 5-10 requests per minute in some cases to undefined limits in other cases.